code/game/objects/structures/crates_lockers/closets/l3closet.dm

```diff
@@ -38,7 +38,7 @@
 	..()
 	new /obj/item/clothing/suit/bio_suit/security(src)
 	new /obj/item/clothing/head/bio_hood/security(src)
-
+	new /obj/item/weapon/gun/energy/taser/xeno/sec(src)

```

Comment:
Weren't there two of these per locker?

Reply:
There were not, one per as one suit per.